### PR TITLE
Add post for tentacle .NET version change

### PR DESCRIPTION
**Pull request by veochen-octopus**
# About

This is a post about an upcoming change to Tentacle where we'll drop support for `.NET Framework 4.5.2` and `.NET Core 3.1`, and move to `.NET Framework 4.8` and `.NET 6`.

# Release

While not 100% tied to a release, it will be bundled with `2023.1`. But the new Tentacle will become available before that.

# Blog image idea

Possibly an image of tentacles holding an icon of `.NET 6` on one "hand", and `.NET Framework 4.8` on another smaller "hand", as the plan is to eventually drop `.NET Framework 4.8`.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [x] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

## How to review this PR

The tone and wording of the post.

## Publication date

As soon as ready. Ideally this month.

### CODE REVIEW

File: blog/2022-q4/tentacle-dotnet-version-change/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+metaImage: ../q4-template-folder/placeholderimg.png
+bannerImage: ../q4-template-folder/placeholderimg.png
+bannerImageAlt: Moving Tentacle from .NET Framework 4.5.2 & .NET Core 3.1 to .NET Framework 4.8 & .NET 6.
+isFeatured: false
+tags:
+  - Product
+  - Tentacle
+---
+
+Due to .NET Framework 4.5.2 being EOL from 26 April 2022, wWe are moving Tentacle to .NET Framework 4.8 for Windows installers only, and .NET 6.0 for everything else including Windows Docker images.
```
Comment (zentron) [inline blog/2022-q4/tentacle-dotnet-version-change/index.md:16]: double `wWe`

File: blog/2022-q4/tentacle-dotnet-version-change/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+In this post, I’ll cover why we’re making this decision now, as well as the most common questions that you might have regarding the Tentacle bump.
+
+
+## Why are we doing this now?
+
+Tentacle had been on `.NET Framework 4.5.2` and `.NET Core 3.1` for some time. The decision to move these versions was taken in light of `.NET Framework 4.5.2` being EOL from 26 April 2022 and `.NET Core 3.1` being EOL from 13 December 2022, which means we'll be missing out on security patches and library updates. In order to ensure we can continue to provide our customers with software that is safe and secure, it is essential that we align with the support schedules of our underlying development frameworks.
```
Comment (zentron) [inline blog/2022-q4/tentacle-dotnet-version-change/index.md:23]: Maybe link back the old post https://octopus.com/blog/tentacle-versioning

File: blog/2022-q4/tentacle-dotnet-version-change/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## What is the plan moving forward?
+
+We are going to compile Tentacle for `.NET 6` as well as `.NET Framework 4.8` rather than only producing builds for `.NET 6`, as there are still a substantial number of deployment targets still using different versions of Windows 7 and 8 - including Windows 7 SP1 and Windows Server 2008 SP2 - which continue to require a supported .NET Framework version in the short to medium term and moving to `.NET 6` would cut off support for these servers. The `.NET 6` builds of Octopus Tentacle will be used for all other platforms, such as Linux packages and Docker Images.
```
Comment (zentron) [inline blog/2022-q4/tentacle-dotnet-version-change/index.md:27]: This sentence feels a little long

File: blog/2022-q4/tentacle-dotnet-version-change/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+No, your deployments will not be affected by this move. By making the incremental move to `.NET Framework 4.8` and `.NET 6`, we remain highly backward compatible for your deployment targets as far back as Windows 7 SP1.
+
+### Should we upgrade deployment targets on Windows 7 SP1 and Windows Server 2008 SP2?
+
+Yes, we encourage you to upgrade your deployment targets that are still on either Windows 7 SP1 or Windows Server 2008 SP2 to a version that supports `.NET Framework 4.8`, ideally a version that is compatible with `.NET 6` and above.
+
+### What if we are unable to upgrade our deployment targets?
+
+Where you are unable to upgrade your deployment targets to a supported .NET version, you’ll need to lock your Tentacle version to ensure it remains functional for those particular deployment targets. To learn more about locking your Tentacle version, please read this [blog post](https://octopus.com/blog/tentacle-versioning) here.
```
Comment (zentron) [inline blog/2022-q4/tentacle-dotnet-version-change/index.md:49]: ... you’ll need to lock your Tentacle version t**o avoid it automatically upgrading** and ensure it remains functional for those particular deployment targets

**Review by TeganAli — APPROVED** [review]

**Pull request merged**